sci-visualization/spectromatic/spectromatic-1.0-r3.ebuild

```diff
@@ -1,25 +1,25 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=8
 
 inherit toolchain-funcs
 
-MY_P=${PN}_${PV}-1
+MY_P="${PN}_${PV}-1"
 
 DESCRIPTION="Generates time-frequency analysis images from wav files"
-HOMEPAGE="http://ieee.uow.edu.au/~daniel/software/spectromatic/"
-SRC_URI="http://ieee.uow.edu.au/~daniel/software/spectromatic/dist/${MY_P}.tar.gz"
+HOMEPAGE="https://wiki.gentoo.org/wiki/No_homepage"
+SRC_URI="mirror://gentoo/${MY_P}.tar.gz"
```

Comment:
This we can't do. The mirrors are a convenience infrastructure, not something we should rely on in an ebuild.

If there really is no proper way to download this software then the package should be removed.

Reply:
Thanks for clarifying - i wasn't sure about that.
I've tried to use http://web.archive.org/web/20080721145215/http://ieee.uow.edu.au/~daniel/software/spectromatic/dist/ as a download link but for some reason the fileformat won't be recognized  when using `wget`, while still working when manually downloading via the browser. (it's still not a perfect solution, but at least it should work)

Generally i've have no strong opinion about treecleaning this ebuild. Since it has no other open bugs i was trying to fix it instead of removing it.

Reply:
I would also think about last-riting in this case.